user_library_sqlite.go

```
@@ -0,0 +1,332 @@
+package minutes
+
+import (
+	"time"
+
+	"github.com/jinzhu/gorm"
+
+	_ "github.com/jinzhu/gorm/dialects/sqlite"
+)
+
+// SqliteUserLibrary is a read-write user-specific library
+type SqliteUserLibrary struct {
+	databaseDir string
+	db          *gorm.DB
+}
+
+// NewSqliteUserLibrary accepts the database directory name and
+// returns a new SqliteUserLibrary instance
+func NewSqliteUserLibrary(DBDir string) *SqliteUserLibrary {
```

`DBDir` should be lower case

---

@superdecimal as we discussed before this should probably return an error.